src/content/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts.mdx

```diff
@@ -166,7 +166,7 @@ Here are some user-related query examples. For details on how users are counted,
 To help [manage your billable data](/docs/telemetry-data-platform/get-data-new-relic/manage-data/manage-your-data), you can set alerts to notify you of unexpected increases in usage. Learn how to create alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
 
 <Callout variant="caution">
-  When creating alert conditions, you should set the [Evaluation offset](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions#evaluation-offset) value to 60 minutes or your conditions may not trigger.
+  When creating alert conditions, you should set the [delay/timer](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions#delay-timer) value to 60 minutes or your conditions may not trigger.
```

Comment:
They will only need to set the delay this high if they are using Cadence, in which case it'll be called Delay, not Timer.
We recommend using Event Timer with this use-case, though, and you can get by with setting the Timer to 30 seconds.